src/components/BookItem.js

```diff
@@ -1,12 +1,21 @@
+/* eslint-disable camelcase */
 import PropTypes from 'prop-types';
 import '../modules/BookItem.css';
-// eslint-disable-next-line arrow-body-style
+import { useDispatch } from 'react-redux';
+import { removedBook } from '../redux/books/bookSlice';
+
 const BookItem = ({ bookListProp }) => {
+  const dispatch = useDispatch();
+  const handleDeleteBook = (item_id) => {
+    dispatch(removedBook(item_id));
+    console.log(item_id);
+  };
```

Comment:
- [ ] It would be nice if you remove the console.log because it shows important data in the console window of the browser which is not good for the security of your app 🤔 So it would be nice if you remove the console.log to follow the JavaScript best practices 👍